stdlib/2/_sre.pyi

```diff
@@ -21,6 +21,7 @@ class SRE_Match(object):
     def groups(self) -> Tuple[Optional[str], ...]: ...
     def span(self) -> Tuple[int, int]:
         raise IndexError()
+    def regs(self) -> Tuple[Tuple[int, int]]: ...  # undocumented
```

Comment:
This type should be `Tuple[Tuple[int, int], ...]`, since the outer tuple can be any length.

(I thought I commented on this in my last review, but I must have forgotten to save the comment or something, sorry!)